pan-domain-auth-play/src/main/scala/com/gu/pandomainauth/action/Actions.scala

```diff
@@ -33,10 +33,10 @@ trait AuthActions {
     */
   def wsClient: WSClient
   def controllerComponents: ControllerComponents
-  def panDomainSettings: PanDomainAuthSettingsRefresher
+  val panDomainSettings: PanDomainAuthSettingsRefresher
 
-  private def system: String = panDomainSettings.system
-  private def domain: String = panDomainSettings.domain
+  private lazy val system: String = panDomainSettings.system
+  private lazy val domain: String = panDomainSettings.domain
```

Comment:
Although `system` & `domain` could have remained as `def`s, the truth is that they are constant `val` members of `PanDomainAuthSettingsRefresher`, so they can not change - consequently, they might as well be `lazy val`s (but they must be `lazy`, as otherwise that would immediately evaluate `panDomainSettings` at construction time, which we want to avoid).